test/ui/segmented-bar/segmented-bar-test.html

```diff
@@ -0,0 +1,144 @@
+                }
+            },
+            "segment2": {
+                "prototype": "montage/ui/segment.reel",
+                "properties": {
+                    "element": {"#": "segment2"}
+                },
+                "bindings": {
+                    "content": {
+                        "<-": "@segmentedBar2:segment.object.{label: region, length: size}"
+                    }
+                }
+            },
+
+
+            "segmentedBar3": {
+                "prototype": "montage/ui/segmented-bar.reel",
+                "properties": {
+                    "element": {"#": "segmentedBar3"},
+                    "childrenPath": "regions",
+                    "contentController": {"@": "treeController"}
+                }
+            },
+            "treeController": {
+                "prototype": "montage/core/tree-controller",
+                "bindings": {
+                    "content": {"<-": "@test.tree"}
+                }
+            },
+            "segment3": {
+                "prototype": "montage/ui/segment.reel",
+                "properties": {
+                    "element": {"#": "segment3"}
+                },
+                "bindings": {
+                    "content": {
+                        "<-": "@segmentedBar3:segment.object.content.{label: region, length: size}"
+                    }
+                }
+            },
+
+
+            "segmentedBar4": {
+                "prototype": "montage/ui/segmented-bar.reel",
+                "properties": {
+                    "element": {"#": "segmentedBar4"},
+                    "contentController": {"@": "rangeController"},
+                    "orientation": "vertical"
+                }
+            },
+            "rangeController": {
+                "prototype": "montage/core/range-controller",
+                "bindings": {
+                    "content": {"<-": "@test.content"}
+                }
+            },
+            "segment4": {
+                "prototype": "montage/ui/segment.reel",
+                "properties": {
+                    "element": {"#": "segment4"}
+                },
+                "bindings": {
+                    "content": {
+                        "<-": "@segmentedBar4:segment.object.{label: region, length: size}"
+                    }
+                }
+            }
+        }
+    </script>
+
+</head>
+<body>
+
+<div style="width: 960px; height: 30px;">
+    <div data-montage-id="segmentedBar1" id="segmentedBar1">
+        <div data-montage-id="segment1"></div>
+    </div>
+</div>
+
+<div data-montage-id="segmentedBar2" id="segmentedBar2">
+    <div data-montage-id="segment2"></div>
+</div>
+
+<div data-montage-id="segmentedBar3" id="segmentedBar3">
+    <div data-montage-id="segment3"></div>
+</div>
+
+<div style="width: 30px; height: 960px;">
+    <div data-montage-id="segmentedBar4" id="segmentedBar4">
+        <div data-montage-id="segment4"></div>
+    </div>
+</div>
+
+</body>
+</html>
```

Comment:
nit: `<body>` contents should be indented, allowing the blank lines around that `<body>` contents to be removed.